scimono-compliance-tests/src/main/java/com/sap/scimono/scim/system/tests/UserOperationsHttpResponseCodeTest.java

```diff
@@ -152,6 +152,18 @@ public void testCreateUserWithEmail201() {
     assertEquals(1, createdUser.getEmails().size(), "Verify emails size");
   }
 
+  @Test
+  @DisplayName("Test create a user with only required attributes and verifies, that the status code is 201")
```

Comment:
Test create a user with only required attributes and verify Http status code: 201